packages/web-console/src/scenes/News/index.tsx

```diff
@@ -0,0 +1,228 @@
+import { Text, Drawer, IconWithTooltip } from "../../components"
+import styled from "styled-components"
+import React, { useEffect, useState, useContext } from "react"
+import { QuestContext } from "../../providers"
+import { NewsItem } from "../../utils/questdb"
+import { useDispatch, useSelector } from "react-redux"
+import { selectors, actions } from "../../store"
+import ReactMarkdown from "react-markdown"
+import { Loader, Button } from "@questdb/react-components"
+import { Notification2 } from "styled-icons/remix-line"
+import { db } from "../../store/db"
+import { UnreadItemsIcon } from "../../components/UnreadItemsIcon"
+import { Thumbnail } from "./thumbnail"
+
+const swing = () => {
+  let animation = ""
+  const numKeyframes = 30 // 5 sec total animation
+
+  for (let i = 0; i < numKeyframes; i++) {
+    const rotateDegree = i % 2 === 0 ? 15 : -15
+    const keyframePercent = (i / numKeyframes) * 100
+    animation += `
+      ${keyframePercent}% {
+        transform: rotate(${rotateDegree}deg);
+      }
+    `
+  }
+  return animation
+}
+
+const BellIcon = styled(Notification2)<{ $unread: boolean }>`
+  color: ${({ theme, $unread }) => theme.color[$unread ? "red" : "foreground"]};
+
+  ${({ $unread }) =>
+    $unread &&
+    `
+  animation: 5s linear swing infinite;
+  @keyframes swing {
+    ${swing()}
+  }
+  `}
+`
```

Comment:
will need to add `keyframes` to `styled-components` import

```suggestion
const swing = keyframes`
  0%,
  50%,
  100% {
    transform: rotate(0);
  }

  10%,
  30% {
    transform: rotate(-5deg);
  }

  20%,
  40% {
    transform: rotate(5deg);
  }
`

const BellIcon = styled(Notification2)<{ $unread: boolean }>`
  color: ${({ theme, $unread }) => theme.color[$unread ? "red" : "foreground"]};

  ${({ $unread }) =>
    $unread &&
    `
  animation: ${swing} 1s infinite ease-in-out;
  transform-origin: 50% 0;
  `}
`
```

Reply:
ignore this, please see #202